core/domain/html_validation_service_test.py

```diff
@@ -852,7 +852,7 @@ def test_validate_customization_args(self):
                 'heading-with-value="&amp;quot;lorem ipsum&amp;quot;">'
                 '</oppia-noninteractive-collapsible>'
             )],
-            "Missing attributes: [u'text-with-value'], Extra attributes: []": [(
+            'Missing attributes: [u\'text-with-value\'], Extra attributes: []': [( # pylint: disable=line-too-long
```

Comment:
Please do not use pylint pragmas. Instead try breaking this up into multiple lines.